{{cookiecutter.repo_slug}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
 <!--- Please provide a general summary of your changes in the title above -->
 
-## Pull request type
+## Pull Request type
```

Comment:
This is a small nitpick. Pull Request is a GitHub-specific term (git itself doesn't have a defined term; other services like GitLab use Merge Request) and so I think it should be capitalized. If you disagree, no big deal :)